### ISSUE TITLE
Staging Review finding: Improper heading structure

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99003]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management
**Product:** Appoint a Representative MVP
**Feature:** Appoint a Representative + Representative Status Widget

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are a number of steps in the workflow where headings aren't correctly nested to reflect the actual structure of the content. Screen reader users often navigate a page by skipping heading to heading, and an appropriate heading structure helps them understand where they are, how to find the next thing, and how different sections are related.

## Recommended action

Problematic heading structure that I noticed includes:
- On /appoint-rep/claimant-type, "Tell us who you are" should be an H2 (nested beneath the H1). Totally okay for this one to be styled as an H3 to match the visual style of subsequent pages, but it needs to be an H2 here.
   - https://github.com/department-of-veterans-affairs/va.gov-team/issues/98029
- On /appoint-rep/representative-select, results card headings should be H4s (nested beneath H3 "Select the accredited rep...")
   - https://github.com/department-of-veterans-affairs/va.gov-team/issues/98030
- [ ] On /appoint-rep/representative-contact, "Contact the accredited representative" should be an H3 (nested beneath the step heading)
- On /appoint-rep/authorize-medical, the accordion trigger "Our records authorization policy" should be an H4 (nested beneath "Authorization to access..."
   - https://github.com/department-of-veterans-affairs/va.gov-team/issues/98033
- On /appoint-rep/authorize-address, the accordion trigger "Our address authorization policy" should be an H4
   - https://github.com/department-of-veterans-affairs/va.gov-team/issues/98034
- [ ] On /appoint-rep/authorize-inside-va, the accordion trigger "Our policy for access through..." should be an H4
- [ ] On /appoint-rep/authorize-outside-va, the accordion trigger "Our policy for access outside..." should be an H4
- On /appoint-rep/review-and-submit, the accordion trigger "Our terms and conditions" should be an H3
   - https://github.com/department-of-veterans-affairs/va.gov-team/issues/98034

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
We have some tickets already that should address this: 

1. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98028
2. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98029
3. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98030
4. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98032
5. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98033
6. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98034
7. https://github.com/department-of-veterans-affairs/va.gov-team/issues/98035
